tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetBase.cs

```diff
@@ -333,7 +333,15 @@ protected async Task TestRateLimiter(bool enableSecurity, string url, MockTracer
             {
                 foreach (var header in headers)
                 {
-                    _httpClient.DefaultRequestHeaders.Add(header.Key, header.Value);
```

Comment:
This is useful to have more control over the headers in test. We can remove default headers by setting them to null or add any desired custom value without the need of adding them to the existing ones.

Reply:
Agree, please log this as a potential improvement.